baseservice/iodevices/base/input_output_devices.py

```diff
@@ -0,0 +1,15 @@
+from abc import ABCMeta
```

Comment:
nononono... why do we need this?! I really wanted to separate input and output devices on this project